lib/v2/scujcc/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/jwb/:category', require('./jwb'));
```

Comment:
https://github.com/DIYgod/RSSHub/blob/92d9c57cbe02e9deb6298642fbeba1616573a01a/lib/v2/scujcc/jwb.js#L16
```suggestion
    router.get('/jwb/:category?', require('./jwb'));
```